parsers/dataField/utils.go

```diff
@@ -37,3 +40,19 @@ func extractTokenAndNonce(arg []byte) (string, uint64) {
 
 	return string(identifier), nonce.Uint64()
 }
+
+func isEmptyAddr(pubKeyConverter core.PubkeyConverter, receiver []byte) bool {
+	emptyAddr := make([]byte, pubKeyConverter.Len())
+
+	return bytes.Equal(receiver, emptyAddr)
+}
+
+func isASCIIString(input string) bool {
```

Comment:
[comment]
Same function has been implemented by @iulianpascalau in logger repo. Could leave a //todo here to move that func in core repo and only keep one function